gcp_variant_transforms/transforms/variant_to_bigquery_test.py

```diff
@@ -101,15 +102,30 @@ def _get_table_schema(self):
     schema.fields.append(call_record)
     return schema
 
+  def _make_header(self, key_num_dict):
```

Comment:
please update this as well.
I now actually think it's better to create a common place that has this implementation and is used in all 3 places. That way, when we switch to using Nucleus, we'd also need to change things in one place. So, please create a new file under `testing/vcf_header_util.py` and make a public `make_header` method in that file.

Reply:
This is still not updated?

Reply:
Sorry, I forgot to delete this local function, otherwise I updated in all those places that it was called. 

It's strange that pylint did not complain about it!